main.go

```diff
@@ -117,8 +127,7 @@ func mainE(config Configs) error {
 		return fmt.Errorf("Failed to open project, error: %s", err)
 	}
 
-	buildTask := gradleProject.
-		GetTask("assemble")
+	buildTask := map[bool]*gradle.Task{true: gradleProject.GetTask("assemble"), false: gradleProject.GetTask("bundle")}[config.BuildType == "apk"]
```

Comment:
For readability, more lines are preferred, using an if.

Reply:
Fixed

Reply:
👍